### DISCUSSION TITLE
Streak Lost even when I did the daily target

### DISCUSSION THREAD

**Discussion by Rafael-Anguiano** [cssbattle/forum#365] [Q&A]
Today I've done my daily target, but it just reset my streak count. 😞😞😞 

![streak lost](https://github.com/cssbattle/forum/assets/77172830/04b12d78-d6ac-4e57-b560-301c4bf70563)

**Comment by yopyx** [comment]
I have the same problem and on top of that there are multiple targets that I get them 100% but for some reason it gives me 99.8 or less

**Comment by Rafael-Anguiano** [comment]
Hi, I noted that the counter restart when the day finished, not when a new daily target is released.
